WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -366,6 +382,28 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForWritingSettingsAt
     return nil;
 }
 
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForDeviceSettingsAtRow:(NSInteger)row
+{
+    switch (row) {
+        case (SiteSettingsDeviceGeotagging):{
+            return self.geotaggingCell;
+        }
+            break;
+        case (SiteSettingsDeviceDefaultCategory):{
+            [self configureDefaultCategoryCell];
+            return self.defaultCategoryCell;
+        }
+            break;
+        case (SiteSettingsDeviceDefaultPostFormat):{
+            [self configureDefaultPostFormatCell];
+            return self.defaultPostFormatCell;
+        }
+            break;
```

Comment:
This is nitpicky code style stuff, but for consistency within the file maybe remove the parenthesis in the `case` statements and remove the `break`s since each case returns?